### DISCUSSION TITLE
how to use with nix develop

### DISCUSSION THREAD

**Discussion by tcoopman** [zellij-org/zellij#2802] [General]
This is the scenario:

I'm running zellij, and start a new tab. Now I want to run this tab in a new `nix develop` environment, so I run: `nix development` in this tab.
The issue is that when I now open a new pane in the tab, it's not running in the `nix develop` env, so I need to remember to run it again.

Is there a way to start a tab and spawn all new panes from the `nix develop`.

**Comment by Nerglej** [comment]
Hello!

I don't know if you've found out yet, but I think a simple solution would be to call `nix develop` *before* running the `zellij` command :smile: 

Have a nice day!

**Reply by tcoopman** [reply]
That's only partly a solution, because it only works if you have 1 directory/ nix environment that you want to load.

As soon as you want to use multiple folders with different shell envs this doesn't work anymore.

Furthermore, it makes the zellij start-up screen kinda useless as well. Because you first need to go to the right directory, and then start zellij.
